firestore/FirestoreSwiftUIExample/Views/WriteReviewView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  WriteReviewView.swift
+//  FirestoreSwiftUIExample
+//
+//  Copyright (c) 2021 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import Firebase
+
+struct WriteReviewView: View {
+  var restaurant: Restaurant
+  @State private var text = "Write a review..."
```

Comment:
Make this initial value a string constant so it can be referenced from one place throughout the file.